src/utils.ts

```diff
@@ -316,6 +315,8 @@ export async function askAndOpenProject(
  */
 export async function askAndOpenInRemote(
     operation: OperationType, telemetryContext: TelemetryContext) {
+  telemetryContext.properties.result = TelemetryResult.Failed;
```

Comment:
Do we still send operation failed telemetry if user select 'Yes' and open the containerized Project successfully?

Reply:
Yes. Because the fail is meant for the former operation, not for the "open in contianer" action.
The former operation may be compile / upload.